src/main/features/claim/views/reason.njk

```diff
@@ -15,6 +15,12 @@
           <label for="{{ name }}" id="{{ name }}[label]">
             <p class="form-label">
               {{ t('Tell us why you believe the defendant owes you money.') }}
+<<<<<<< HEAD
```

Comment:
conflict

Reply:
for some reason it won't let me try to resolve this conflict, as it normally does on these PRs.